File: Models/AngleCalibrationTracePoint.vb
Namespace Models
    Public Class AngleCalibrationTracePoint
        Private ReadOnly _x As Double
        Private ReadOnly _y As Double

        Public Sub New(x As Double, y As Double)
            _x = x
            _y = y
        End Sub

        Public ReadOnly Property X As Double
            Get
                Return _x
            End Get
        End Property

        Public ReadOnly Property Y As Double
            Get
                Return _y
            End Get
        End Property
    End Class
End Namespace

File: src/ClickSyncMouseTester/Models/AngleCalibrationTracePoint.cs
namespace ClickSyncMouseTester.Models;

public class AngleCalibrationTracePoint
{
    private readonly double _x;

    private readonly double _y;

    public double X => _x;

    public double Y => _y;

    public AngleCalibrationTracePoint(double x, double y)
    {
        _x = x;
        _y = y;
    }
}





